### ISSUE TITLE
Language toggle

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/report-a-cybercrime#63]
Manual testing with a keyboard + voice recognition I am unable to active the language element. 
A screen reader user may be able to access this by pulling up a list of links/buttons on the page, but a keyboard user will be unable to do so.
